### DISCUSSION TITLE
COMPILATION ERRORS.

### DISCUSSION THREAD

**Discussion by qveezzx** [CosmosOS/Cosmos#3069] [General]
not sure if it should be here but:
![image](https://github.com/user-attachments/assets/e564a6d1-6261-443d-bca3-dff073c15393)

Have you checked Github Issues for similar errors?
yes, didnt find anything.

**Exception**
Post the exception returned by Visual Studio
^^

**Visual Studio Output Logs**
Post the entire output log given by Visual Studio for the build
^^

**Screenshots**
^^

**Context**
Before posting please confirm that the following are in order
[ ✅] Both Cosmos VS Extensions are installed
[ ✅ ] In the NuGet Package Manager "Include prerelease" is selected
[ ✅ ] The Cosmos NuGet package store is selected (NOT nuget.org) in 'Manage NuGet Packages'
[ ✅ ] The Cosmos NuGet packages are installed

Add any other context about the problem which might be helpful.
its "Cannot implicitly convert type 'long' to 'int'" tho

**Comment by tails1154** [comment]
put (int) next to the numbers may do it. Most likely not though.

**Reply by qveezzx** [reply]
thank you, i forgot to close issue but it was fixed long time ago tho, still thank you

**Comment by Szymekk44** [comment]
ooh my RadianceOS here... i changed vfs disk and partition class so now use (long) after all operations.
I do not recommend making your own os from it, because its old and junky. You can you my cosmos optimization kit. There is a great base, window system and of course optimization

**Reply by qveezzx** [reply]
yeah, i fixed it just like tails2012... said
